apps/music/js/metadata/vorbis_picture.js

```diff
@@ -0,0 +1,134 @@
+/* global BlobView */
```

Comment:
I don't mind refactoring. But step by step at that point. Multi page is coming in a different PR (it works)